package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "node ./scripts/lint.js --cache --ext .js,.ts,.tsx --max-warnings 0",
     "linc": "node ./scripts/linc.js --cache --fix --ext .js,.ts,.tsx --max-warnings 0",
     "modular": "ts-node packages/modular-scripts/src/cli.ts",
-    "test": "yarn modular test",
+    "test": "yarn modular test --watchAll=false --runInBand",
```

Comment:
Rather than have this - potentially we just always add ``--runInBand`` into the `test` command when we're running with `CI=true` in the environment?

Reply:
We don't want that for every modular project, I'd think, it'll make test suites slow. We only only want it for our own because we mess with the file system and stuff which breaks snapshot tests because they may not be stable when running them all together.